app/src/signing/networkMessages/polkadot-transactions.js

```diff
@@ -34,6 +44,7 @@ export async function getAPI(endpoint) {
 // }
 
 export async function multiMessage(transactions) {
+  const polkadotNetworks = await getPolkadotNetworks()
```

Comment:
this would be called every time. why not just get the normal networks list?

Reply:
from where? There is no normal networks list, only in the store's state... But not sure how that would work out here

Reply:
we only need the rpc_urls right? let's go with just a dictionary for the rpc_urls

Reply:
but we need to update them no? What happens if they suddenly change? Then we need to remember updating them here too

Reply:
True! We could pass public rpc urls independent from our internal rpc urls. But send both from the Api

Reply:
You mean creating a query for rpcUrls specifically? Isn't is simpler to just stick to the current networks query?

Reply:
No I mean returning the public endpoints from the API for the networks query. We could define a property in the networks "public_rpc_endpoint" and only return that.

Reply:
But why not just return the rpc_urls? I think I'm missing something here. Will keep looking at the commits